inc/constants.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+/* ====== LIBRARIES ====== */
+
+/* ====== DECLARATIONS ====== */
+
+namespace constants {
+    extern const int g_decimalBase;
```

Comment:
Here is 4 spaces after namespace opening, indenting the global once

But it shouldn't have an indent: 
in .clang-format the setting is `NamespaceIndentation: Inner`
This causes inner namespaces (a namespace in a namespace) to be indented, see [documentation](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#namespaceindentation)
Since we only have one namespace constants it should be formatted to 
```
namespace constants {
// i like an extra line here
const int g_decimalBase = 10;

} // constants
```
This is also what I get when I autoformat the file (except the extra line after namespace)

See also utilities{.hpp, .cpp} where its formatted the same way

Reply:
Oh, inside the namespace in front of its members. Thought you meant after the word "namespace" itself. Changed.